gto/config.py

```diff
@@ -26,6 +26,10 @@ def check_name_is_valid(name):
     return bool(re.match(r"[a-z][a-z0-9-/]*[a-z0-9]$", name))
 
 
+def check_requirement_is_valid(name):
+    return bool(re.match(r".+:\s\d", name))
```

Comment:
What should be the expected format for `--requirements`? 
Should we enforce a pattern or not?

I am forcing `{name}: {version}` here